core/network/src/main/java/org/sopt/network/model/response/base/ApiError.kt

```diff
@@ -0,0 +1,15 @@
+package org.sopt.network.model.response.base
+
+import kotlinx.serialization.Serializable
+import org.sopt.model.Base
+
+@Serializable
+data class ApiError(
+    val code: Int,
+    val message: String,
+)
```

Comment:
```suggestion
@Serializable
data class ApiError(
    val code: Int,
    val message: String,
) : Exception()
```

이런걸 상속받아서 Custom Exception 형태로 제작하면 runCatchong에서 해당 Error(Excpetion)을 throw할 수 있습니다.

Reply:
감사합니다 ! custom Exception 활용해서 에러처리를 하면 보다 여러 상황에 유연하게 대응할 수 있을 것 같습니다! 적용해보겠습니다 !